lib/mix/lib/mix/tasks/app.tree.ex

```diff
@@ -20,6 +20,8 @@ defmodule Mix.Tasks.App.Tree do
     * `--pretty` - use Unicode codepoints for formatting the tree.
       Defaults to true except on Windows.
 
+    * `--dot` - produces a DOT graph description of the application tree.
```

Comment:
Definitely considered this, @mootpointer suggested the same thing as there are likely to be other formats for this. What are others thoughts?